packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -1954,6 +1954,32 @@ describe('cluster', () => {
     });
   });
 
+  test('create a cluster from a snapshot with encrypted storage', () => {
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseClusterFromSnapshot(stack, 'Database', {
+      engine: DatabaseClusterEngine.aurora({ version: AuroraEngineVersion.VER_1_22_2 }),
+      instanceProps: {
+        vpc,
+      },
+      snapshotIdentifier: 'mySnapshot',
+      storageEncryptionKey: kms.Key.fromKeyArn(stack, 'Key', 'KeyArn'),
```

Comment:
Pretty sure you need an actual ARN here 😉. See [here](https://github.com/aws/aws-cdk/blob/19ead77c6470c0aca4900c4ee200a7c4e66ebbed/packages/%40aws-cdk/aws-codepipeline-actions/test/codecommit/codecommit-source-action.test.ts#L533-L534).